unbxdSearch.js

```diff
@@ -1039,20 +1135,21 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
 
             return this;
         },
-        isUsingPagination: function() {
+        isUsingPagination: function () {
             return !this.options.isAutoScroll && this.options.isPagination;
         },
-        getHostNPath: function() {
+        getHostNPath: function () {
             var handler = "search";
             if (isTypeBrowseOrCategory(this.options.type)) {
                 handler = this.options.type;
             }
-            return "//search.unbxdapi.com/" + this.options.APIKey + "/" + this.options.siteName + "/" + handler;
+            var searchEndPoint = this.options.searchEndPoint || "https://search.unbxd.io";
```

Comment:
this should backward compatible with COM: `https://search.unbxdapi.com/`. So we can use this directly while bundling.

Reply:
We removed https://search.unbxdapi.com/ because this is for new customer integrations only.

Reply:
If we are publishing this, we need to support both COM and IO. If you upload this to CDN we need backward compatibility unless we add version on fileNames.


We have customers who still uses unbxdSearch.js present in the CDN.
https://github.com/unbxd/customer-js/blob/master/liquidationchannel/js/search.js